jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -254,27 +254,32 @@ def mos_spec2d_parser(app, data_obj, data_labels=None, add_to_table=True,
         the mosviz table.
     data_labels : str, optional
         The label applied to the glue data component.
+    extension: int, optional
+        The extension in the fits file that contains the data to be loaded.
+    transpose: bool, optional
```

Comment:
When should one use this?
```suggestion
    transpose : bool, optional
```